src/main/java/com/patchworkmc/mapping/remapper/ManifestRemapper.java

```diff
@@ -23,21 +24,30 @@ public String remapMemberDescription(String descriptor) {
 
 	@Override
 	public String remapFieldName(String owner, String name, String descriptor) {
-		return asmRemapper.mapFieldName(owner, name, descriptor);
+		if (descriptor.equals("")) {
+			// You would think ignoreFieldDesc would work instead, but it doesn't. No idea why.
```

Comment:
Perhaps ignoreFieldDesc only works when you pass `null` as opposed to an empty string like the code previously did.

Reply:
Didn't work.